src/bindable-property.js

```diff
@@ -58,17 +64,55 @@ export class BindableProperty {
   * @param descriptor The property descriptor for this property.
   */
   registerWith(target: Function, behavior: HtmlBehaviorResource, descriptor?: Object): void {
+    let { reflect } = this;
     behavior.properties.push(this);
     behavior.attributes[this.attribute] = this;
     this.owner = behavior;
 
+    if (reflect) {
+      behavior._registerReflection(this.name, typeof reflect === 'function' ? reflect : propToAttr);
+      this._configureReflection(target);
+    }
+
     if (descriptor) {
       this.descriptor = descriptor;
       return this._configureDescriptor(descriptor);
     }
 
     return undefined;
   }
+  
+  _configureReflection(target) {
+    if (target.__reflectionConfigured__) return;
```

Comment:
Aurelia includes a `Symbol` polyfill.
Is the `__xxx__` field how we want to roll when hacking into objects we don't own? Shouldn't we use symbols instead?

Reply:
Some thing I couldn't think of. Can only 👍